<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>jatekosAdatok</class>
 <widget class="QWidget" name="jatekosAdatok">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel
{
	font: 14pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(255, 255, 255);
}</string>
  </property>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>301</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold 18pt &quot;Poor Richard&quot;;
color: rgb(255, 255, 255);
/*padding-left: 160px;
padding-top: 50px;
padding-bottom: 30px;*/</string>
   </property>
   <property name="text">
    <string>Alapadatok</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignHCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="playerName">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>119</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">padding-right: 30px;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>126</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Felhasználónév</string>
   </property>
  </widget>
  <widget class="QLabel" name="userFullName">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>170</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>168</y>
     <width>142</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Felhasználó neve</string>
   </property>
  </widget>
  <widget class="QLabel" name="userEmail">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>210</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Email cím</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
